README.md

```diff
@@ -131,14 +134,18 @@ object MySuite extends IOSuite {
 
 #### Other suites
 
-Weaver also includes support for `ZIO`-based suites via the optional `weaver-zio` dependency.
+Weaver also includes support for 
```

Comment:
Thanks for the added bits in the readme. Could you also add a page in the website ? You'll have to : 

* add a new markdown file like [this one]((https://github.com/bfdes/weaver-test/blob/monix-support/docs/zio_usage.md) in the same directory
* reference it  [there](https://github.com/bfdes/weaver-test/blob/master/website/sidebars.json#L15 )

You can build the website locally following these [instructions](https://scalameta.org/mdoc/docs/docusaurus.html#create-static-site) and call `yarn run start` to automatically open it in your browser

Reply:
Addressed in 4b29674bf0724234761a5a08f6337557ec19de7f.